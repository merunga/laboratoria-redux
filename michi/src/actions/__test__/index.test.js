import * as actions from '../'

describe('actions', () => {
  it('selecciona casillas', () => {
    const row = [0, 1, 2]
    /* eslint-disable array-callback-return */
    row.map((i) => {
      row.map((j) => {
        expect(actions.select(i, j)).toMatchSnapshot()
      })
    })
    /* eslint-enable array-callback-return */
  })
  it('reinicia juego', () => {
    expect(actions.reset()).toMatchSnapshot()
  })
})
