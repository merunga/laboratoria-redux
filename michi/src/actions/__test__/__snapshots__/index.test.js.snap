// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actions reinicia juego 1`] = `
Object {
  "type": "RESET",
}
`;

exports[`actions selecciona casillas 1`] = `
Object {
  "type": "SELECT",
  "x": 0,
  "y": 0,
}
`;

exports[`actions selecciona casillas 2`] = `
Object {
  "type": "SELECT",
  "x": 0,
  "y": 1,
}
`;

exports[`actions selecciona casillas 3`] = `
Object {
  "type": "SELECT",
  "x": 0,
  "y": 2,
}
`;

exports[`actions selecciona casillas 4`] = `
Object {
  "type": "SELECT",
  "x": 1,
  "y": 0,
}
`;

exports[`actions selecciona casillas 5`] = `
Object {
  "type": "SELECT",
  "x": 1,
  "y": 1,
}
`;

exports[`actions selecciona casillas 6`] = `
Object {
  "type": "SELECT",
  "x": 1,
  "y": 2,
}
`;

exports[`actions selecciona casillas 7`] = `
Object {
  "type": "SELECT",
  "x": 2,
  "y": 0,
}
`;

exports[`actions selecciona casillas 8`] = `
Object {
  "type": "SELECT",
  "x": 2,
  "y": 1,
}
`;

exports[`actions selecciona casillas 9`] = `
Object {
  "type": "SELECT",
  "x": 2,
  "y": 2,
}
`;
